src/util/psbt/map/global.rs

```diff
@@ -203,7 +203,7 @@ impl PartiallySignedTransaction {
         if let Some(tx) = tx {
             Ok(PartiallySignedTransaction {
                 unsigned_tx: tx,
-                version: version.unwrap_or(0),
+                version: Version::from_standard(version.unwrap_or(Version::PsbtV0 as u32)).map_err(Error::from)?,
```

Comment:
Perhaps this is cleaner
```rust
        let version = match version {
            Some(v) => Version::from_standard(v).map_err(Error::from)?,
            None => Version::PsbtV0, // Should be Version::default()
        };
```
Then use plain old `version,` to set the field in PSBT.